icubam/backoffice/server.py

```diff
@@ -78,6 +78,13 @@ def make_routes(self, path):
     self.add_handler(messages.ListMessagesHandler)
     self.add_handler(maps.MapsHandler)
 
+    if os.path.isdir(self.config.backoffice.extra_plots_dir):
+      route = os.path.join("/", self.root, r'static/extra-plots/(.*)')
```

Comment:
Were you able to check this works with self.root != ''?

Reply:
Yes, locally for me paths looks like `/bo/static/extra-plots/lineplot_beds_grand_est.png` where root is `/bo`